projects/plugins/jetpack/changelog/add-open-ai-useselect

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: bugfix
+
+Fix openAI paragraph
```

Comment:
Perhaps we should make this more specific?

Reply:
This PR adds some improvements as well?

Reply:
Done